airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -96,30 +102,31 @@ def poll_job_completion(self):
     def execute(self, context):
         # create job_yaml_string from the parameter dictionary.
         # append extra parameters to commands inside the job
-        if not self.job_yaml_string and self.job_param_dict and 'containers' in self.job_param_dict:
+        instance_param_dict = self.job_param_dict.copy()
```

Comment:
Why do we need a copy of self.job_param_dict?  This is not used in any other case.